SSW.Rules.GPT.sln

```diff
@@ -31,6 +31,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Application.UnitTests", "te
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Application.IntegrationTests", "tests\Application.IntegrationTests\Application.IntegrationTests.csproj", "{8F5EA9E9-6B28-4308-B8D0-A0C28E05204D}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SharedClasses", "src\SharedClasses\SharedClasses.csproj", "{F86915AF-37EA-4B49-987A-3AE0978D1EBA}"
```

Comment:
I don't think the name 'classes' adds much value.  Can we just call this project 'Shared' instead?